package.json

```
@@ -1,10 +1,10 @@
 {
   "name": "@iexec/poco",
-  "version": "5.5.0",
+  "version": "5.6.0-rc1",
```

```suggestion
  "version": "5.6.0-rc",
```
based on semver https://semver.org/lang/fr/